hw/dv/tools/dvsim/xcelium.hjson

```diff
@@ -21,6 +21,9 @@
                "-snapshot {tb}",
                // for uvm_hdl_* used by csr backdoor
                "-access +rw",
+               // This is to fix a timescale issue due to using `resetall to reset timescale and
+               // `-sv` compile option. #7178
```

Comment:
This commented out line appears to not be needed? Note the `-sv` is already added on line 8.

Reply:
perhaps my comment is not very clear.
Since we use `-sv` and resetall, we need to add following option to avoid the timescale issue.

Reply:
I think it is clear enough

Reply:
Oops, i read the second comment line as a commented out build switch. nvm.